packages/forms/src/Components/Builder.php

```diff
@@ -225,6 +227,13 @@ public function disableItemDeletion(bool | Closure $condition = true): static
         return $this;
     }
 
+    public function contained(bool | Closure $condition = true): static
```

Comment:
Maybe we should call this feature "inset"? "Contained" sounds not clear enough

Reply:
Sounds good